src/api/app/models/watched_item.rb

```diff
@@ -2,7 +2,8 @@ class WatchedItem < ApplicationRecord
   belongs_to :watchable, polymorphic: true
   belongs_to :user
 
-  validates :user_id, uniqueness: { scope: [:watchable_type, :watchable_id] }
+  validates :watchable_type, :watchable_id, :presence => true
```

Comment:
According to [this cop](https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsredundantpresencevalidationonbelongsto), we don't need to validate the presence